src/pages/loginPage.tsx

```
@@ -0,0 +1,13 @@
+import { useNavigate } from 'react-router-dom';
+
+export const Login = () => {
+    const navigate = useNavigate();
+
+    return (
+        <div>
+            <h1>Welcome!</h1>
+            <p>Click Login to view the dashboard.</p>
+            <button onClick={() => void navigate('/overview')}>Login</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)